src/main/java/menu/MyMenu.java

```diff
@@ -0,0 +1,41 @@
+package menu;
+
+/*
+* To add al the main menu list
+* 1. Manage Students
+* 2. Training Schedule
+* 3. Training Circuits
+*
+ */
+public class MyMenu {
+    private String myCategory;
+    private String menu;
+
+
+    public MyMenu(String myCategory, String menu) {
+        this.myCategory = myCategory;
+        this.menu = menu;
+    }
+
+
+    public String getMyCategory() {
+        return myCategory;
+    }
+
+    public String getMenu() {
+        return menu;
+    }
+
+//    public void trainingScheduleHeading() {
+//        System.out.flush();
+//        System.out.println("TRAINING SCHEDULE: \n");
+//    }
+//    public void manageStudentsHeading() {
+//        System.out.flush();
+//        System.out.println("MANAGE STUDENTS: \n");
+//    }
+//    public void trainingProgramHeading() {
+//        System.out.flush();
+//        System.out.println("TRAINING PROGRAM: \n");
+//    }
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. Now that we are using a revision control system, we can recover any deleted code later.